docs/pages/intro/what-is-compositor.md

```diff
@@ -1,8 +1,13 @@
 # What is Live Compositor?
 
-LiveCompositor is a media server for real-time, low latency, programmable video and audio mixing.
+Live Compositor is an engine for applying effects to videos and for combining multiple videos together.
```

Comment:
`engine` sound wierd. IMO media server is more informative.

Reply:
I actually think the opposite. I think most people who heard something about media servers did so in the context of Jellyfin or Plex, or maybe Janus. I think this can be a source of confusion, since Live Compositor is so much different from these media servers. I don't think people realize how wide the meaning is.

Besides the potential confusion, I don't see the benefit of calling it a 'media server'. How is that more informative, if we in the next sentence explain that it receives information via an http api and you can stream videos in and out? It doesn't add any new information, after reading the next sentence people know it's a server.

I used the word 'engine' very deliberately, because it suggests the compositor should be used in tandem with another thing that drives it.